UnitySDK/Assets/ML-Agents/Editor/Tests/MLAgentsEditModeTest.cs

```diff
@@ -344,6 +335,11 @@ public void TestAcademy()
                 }
 
                 stepsSinceReset += 1;
+<<<<<<< HEAD
```

Comment:
Looks like something went wrong with a merge?

Reply:
Good catch!

Reply:
There's a pre-commit check we can add for this. Remind me later.